### PR TITLE
List exams for Role Users

### PR DESCRIPTION
**Pull request by minhpham11**
## Checklist
- [ ] This is bug/feature (delete incorrect one)
- [ ] Title including ticket number and title

## What this PR do?
Explain clearly and briefly what this PR does. Make a meaningful sentence instead of "this PR solve ticket #xxx"

## Context/Background
More information on the issue that you want to provide, related issues and PRs you have to write information to ask orally or in informal place too.

## Screenshots / Demo & Instruction for testing (if applicable)
image
- link to demo
- instruction on how to test this

## References
- Blog posts / web pages relevant to this issue (which you get your solution / idea from and it may help others to understand the issue quickly)
- Do we need update our knowledge base / project wiki?

### CODE REVIEW

File: app/assets/javascripts/common.js [hunk 1]
```diff
@@ -1,3 +1,105 @@
...
+  //       <button type="button" class="timer__btn timer__btn--reset">
+  //         <span class="material-icons">timer</span>
+  //       </button>
+  //     `;
+  //   }
+  // }
+
+  // new Timer(
+  //   document.querySelector(".timer")
+  // );
```
Comment (rikai-hai) [inline app/assets/javascripts/common.js]: ko dùng thì xóa hết

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -499,4 +499,38 @@ input {
...
+    //     display: inline-block;
+    //     padding: 24px 32px;
+    //     border-radius: 30px;
+    //     background: white;
+    //     position: absolute;
+    //     z-index: 2;
+    // }
+
+    // .timer__part {
+    //     font-size: 36px;
```
Comment (rikai-hai) [inline app/assets/stylesheets/custom.scss]: same above

File: app/controllers/user/exams_controller.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+  def index
+    @exams = Exam.order_by_name.paginate(page: params[:page], per_page: Settings.paginate.manage)
+  end
+
+  def edit
+  end
+
+  def show
+    @questions = Question.includes(:exam).where(exam_id: @exam.id).order_by_content
```
Comment (rikai-hai) [inline app/controllers/user/exams_controller.rb]:
```suggestion
    @questions = @exam.questions.order_by_content
```

Comment (minhpham11) [reply app/controllers/user/exams_controller.rb]: Sử dụng include để hạn chế n+1 cho hệ thống, sao lại suggest cái này.

**Pull request merged**